rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1404,6 +1471,34 @@ TensorList *MasterGraph::labels_meta_data() {
     return &_labels_tensor_list;
 }
 
+std::vector<rocalTensorList *> 
+MasterGraph::ascii_values_meta_data() {
+    if (_external_source_reader) {
```

Comment:
Is this tested? What is done here?

Reply:
No, this is not tested with external source reader.

Reply:
So do we need this condition?